invenio_administration/assets/semantic-ui/js/invenio_administration/src/formik/AdminForm.js

```diff
@@ -69,6 +70,7 @@ export class AdminForm extends Component {
 
   onCancel = () => {
     const { resource } = this.props;
+    console.log(resource);
```

Comment:
👀

Reply:
Just want to say that I thought I was changing this file in a different branch.